Jenkinsfile

```diff
@@ -6,116 +6,115 @@ pipeline {
+                                                checkout scm
+                                                powershell '& ./make.ps1 -Build ' + image + ' test'
+                                                junit(allowEmptyResults: true, keepLongStdio: true, testResults: 'target/**/junit-results.xml')
+                                            }
+                                        }
+                                    }
+                                    // Peform the parallel execution
+                                    parallel parallelBuilds
+                                }
+                            }
+                        }
+                        stage('Deploy to DockerHub') {
+                            // This stage is the "CD" and should only be run when a tag triggered the build
+                            when {
+                                buildingTag()
+                            }
+                            steps {
+                                script {
+                                    // This function is defined in the jenkins-infra/pipeline-library
+                                    infra.withDockerCredentials {
+                                        powershell "& ./make.ps1 -PushVersions -VersionTag $tagName publish"
+                                    }
+                                }
                             }
                         }
                     }
                 }
-            }
-            post {
-                always {
-                    junit(allowEmptyResults: true, keepLongStdio: true, testResults: 'target/*.xml')
-                }
-            }
-        }
-
-        stage('Windows') {
-            matrix {
-                axes {
-                    axis {
-                        name 'IMAGE_NAME'
-                        values 'jdk11-windowsservercore-ltsc2019', 'jdk11-nanoserver-1809', 'jdk17-windowsservercore-ltsc2019', 'jdk17-nanoserver-1809'
+                stage('Linux') {
+                    agent {
+                        label "docker&&linux"
                     }
-                }
-
-                stages {
-                    stage('Build') {
-                        agent {
-                            label 'docker-windows'
-                        }
-                        options {
-                            timeout(time: 60, unit: 'MINUTES')
-                        }
-                        environment {
-                            DOCKERHUB_ORGANISATION = "${infra.isTrusted() ? 'jenkins' : 'jenkins4eval'}"
+                    options {
+                        timeout(time: 30, unit: 'MINUTES')
+                    }
+                    environment {
+                        JENKINS_REPO = "${infra.isTrusted() ? 'jenkins' : 'jenkins4eval'}/inbound-agent"
+                    }
+                    stages {
+                        stage('Prepare Docker BuildX Runner for multi-arch') {
```

Comment:
shorter stage name please, not readable in blue ocean

Reply:
Done.